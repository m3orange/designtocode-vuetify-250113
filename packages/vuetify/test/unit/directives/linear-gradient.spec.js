import Vue from 'vue'
import { test } from '@/test'
import LinearGradient from '@/directives/linear-gradient'

test('VLinearGradient', ({ mount }) => {
  it('LinearGradient should render element with linear gradient', () => {
    const testComponent = Vue.component('test', {
      directives: {
        LinearGradient
      },
      render (h){
        const data = {
          directives: [{
            name: 'linear-gradient',
            value: {
              angle: 45,
              stops: [
                {
                  color: "#42f4e5",
                  percent: 20
                },
                {
                  color: "#409ced"
                }
              ]
            }
          }],
          style: {
            width: "200px",
            height: "200px"
          }
        }
        return h('div', data)
      }
    })

    const wrapper = mount(testComponent)

    expect(wrapper.html).toMatchSnapshot()
  })
})
