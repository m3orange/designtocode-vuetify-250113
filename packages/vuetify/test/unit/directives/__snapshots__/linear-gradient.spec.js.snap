// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VLinearGradient LinearGradient should render element with linear gradient 1`] = `[Function]`;
